src/state/notifications/actions.ts

```diff
@@ -249,7 +251,7 @@ export const unsubscribeWallet: UnsubscribeWalletActionCreator = (wallets, email
   payload: { wallets, email },
 });
 
-export const unsubscribeWalletSuccess = (sessionToken: string): SubscribeWalletSuccessAction => ({
+export const unsubscribeWalletSuccess = (result: Result, sessionToken: string): SubscribeWalletSuccessAction => ({
```

Comment:
result is unused, should be removed from this action

Reply:
Done